services/src/test/resources/config-gui.xml

```diff
@@ -0,0 +1,315 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
It is for mocking the server. I am cleaning all this test resources files now and try to make the unit tests without mocks.